UI/ui_item.csv

```diff
@@ -634,7 +634,7 @@ UseItemBagMessage#0,"""This will unlock \""$(0)\""<br>and acquire the following
 UseItemBagError1#0,"""Insufficient space in inventory to receive items."""
 UseItemBagError2#0,"""Exceeded the max number of items that you can carry."""
 UseItemBagError3#0,"""Failed to acquire item."""
-ConsumeErrorMessage12#0,"""効果時間の上限を超えるため、<br>同じアイテムは使用できません。"""
+ConsumeErrorMessage12#0,"""You cannot use this item as<br>it would exceed the duration limit.""""
```

Comment:
Why there an extra quote here?

Reply:
Because I was too tired, lol. I'll fix it, danke.